### PR TITLE
Add py-graphviz

### PR DESCRIPTION
**Pull request by corburn**
Move non-bio recipes from bioconda to conda-forge.
bioconda/bioconda-recipes#10527

### CODE REVIEW

File: recipes/py-graphviz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.python.org/packages/source/g/graphviz/graphviz-{{ version }}.zip
+  sha256: '{{ sha256 }}'
+
+build:
+  noarch: python
+  number: 0
+  script: $PYTHON -m pip install --no-deps --ignore-installed --no-cache-dir -vvv .
```
Comment (bgruening) [inline recipes/py-graphviz/meta.yaml]: mh, just try `python` here

File: recipes/py-graphviz/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+about:
+  home: https://github.com/xflr6/graphviz
+  license: MIT
+  license_family: MIT
+  summary: Simple Python interface for Graphviz
+
+extra:
+  recipe-maintainers:
+    - corburn
```
Comment (bgruening) [inline recipes/py-graphviz/meta.yaml]: feel free to add me if you need help

File: recipes/py-graphviz/meta.yaml (hunk 1 above)
Comment (corburn) [reply recipes/py-graphviz/meta.yaml]: @bgruening thanks for the review.

I assume `python` `$PYTHON` and `{{ PYTHON }}` differ only in when the PATH is evaluated and using `python` is a matter of preference.

https://github.com/conda-forge/staged-recipes/blob/f46dacf1e9451f6c97d4f603a1bfaa0b78cadf99/recipes/example/meta.yaml#L33

File: recipes/py-graphviz/meta.yaml (hunk 2 above)
Comment (corburn) [reply recipes/py-graphviz/meta.yaml]: @bgruening happy to add you

**Comment by mbargull** [comment]
Looks like this didn't create a feedstock, yet. Which is actually a good thing here :laughing:. In `defaults` as well as `conda-forge` we have `python-graphviz` which is exactly this package. If anyone of you two want to add yourself as a maintainer [over at that feedstock](https://github.com/conda-forge/python-graphviz-feedstock), feel free to do so :+1:.

@conda-forge/core: Since the feedstock isn't created by now, what is the best way to prevent the duplication?

**Comment by jakirkham** [comment]
Doing this with PR ( https://github.com/conda-forge/staged-recipes/pull/6534 ).

**Comment by mbargull** [comment]
Thanks @isuruf and @jakirkham!

**Comment by bgruening** [comment]
Upps, sorry. Thanks for cleaning up!

**Pull request merged**